installing/version-scheme.md

```diff
@@ -28,13 +28,13 @@ Release version
 ---------------
 
 Qubes OS as a whole is released from time to time. Version scheme for all
-releases is modeled after Linux kernel version numbers. When announcing new
+releases is modeled after Linux kernel version numbers. When announcing a new
 release, we decide on the major.minor version (like `3.0`) and release
 `3.0-rc1`. When we feel that enough progress has been made, we put `3.0-rc2`
 and so on. All these versions are considered unstable and not ready for
 production. You may ask for support on mailing lists (specifically
 **qubes-devel**), but it is not guaranteed (you may for example get answer
-„update to newer `-rc`”). Public ISO image may or may not be available.
+“update to newer `-rc`”). Public ISO image may or may not be available.
```

Comment:
https://github.com/QubesOS/qubes-issues/issues/1667 and https://github.com/QubesOS/qubes-posts/commit/4ef0efd81a2e40035346894b246db946b7524dfc and this. Hmm... maybe I should add an automatic check for this

xref https://github.com/QubesOS/qubes-issues/issues/2725